### PR TITLE
OWNERS update

### PR DESCRIPTION
**Pull request by paulfantom**
- Removing Krasi and Bartek from OWNERS file
- Adding Vasily as a maintainer

Related to #3367

### CODE REVIEW

**Comment by brancz** [comment]
Can we add @vsliouniaev here as well?

File: OWNERS [hunk 1]
```diff
@@ -1,7 +1,6 @@
 reviewers:
 - brancz
 - bwplotka
```
Comment (bwplotka) [inline OWNERS:3]: Feel free to remove me from now, I was not very active the last half-year. If I will have more contributions and there will be value in having me as a maintainer going forward we can vote then. (:

**Comment by paulfantom** [comment]
Added @vsliouniaev :)

File: OWNERS (hunk 1 above)
Comment (paulfantom) [reply OWNERS]: Removed :crying_cat_face:

**Comment by lilic** [comment]
@vsliouniaev would you be interested in doing the next release?

**Pull request merged**